app/test/model/post.test.js

```diff
@@ -72,3 +72,34 @@ test('add post and use isSlugExist function to find out if the slug was unique',
     await postModel.deletePost(post1);
     expect(await postModel.isSlugExist(post1)).toEqual(false);
 })
+
+test('check getting all posts', async () => {
```

Comment:
jeśli w bazie danych znajdują się inne posty niż te tutaj to test przechodzi?

Reply:
<facepalm> :-(

Reply:
to znaczy że nie? ;p

Reply:
jak najbardziej tak ;P